src/settings.py

```diff
@@ -82,6 +82,24 @@ def load(path: str):
             f"[TELEGRAM] '{e}' is missing in config. Set Telegram to False")
         SEND_EMAIL = False
 
+    try:
+        SEND_APPRISE = True if config["APPRISE"]["enable"].lower() == "true" else False
+
+    except KeyError:
+        log.warning(
+            "[APPRISE] 'enable' is missing in Config. Set False")
+        SEND_APPRISE = False
+
+    try:
+        if SEND_APPRISE:
+            APPRISEOBJ = apprise.Apprise()
```

Comment:
The settings.py should only contain constant default types. Can you move the creation of the apprise object to the send_apprise function in alerts.py?